metricbeat/module/windows/services/_meta/data.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Can you add a `TestData` function ([example](https://github.com/elastic/beats/blob/65ea8cf4ff50573fc18a9f9224f6e8e37a19fc25/metricbeat/module/system/uptime/metricset_test.go#L11-L28)) to the package that will generate this file when you run `go test -data .` from within this package's directory.